opensearch/src/main/java/org/opensearch/sql/opensearch/response/OpenSearchResponse.java

```diff
@@ -48,27 +48,35 @@ public class OpenSearchResponse implements Iterable<ExprValue> {
   /** List of requested include fields. */
   private final List<String> includes;
 
+  private boolean fieldTypeTolerance;
+
   /** OpenSearchExprValueFactory used to build ExprValue from search result. */
   @EqualsAndHashCode.Exclude private final OpenSearchExprValueFactory exprValueFactory;
 
   /** Constructor of OpenSearchResponse. */
   public OpenSearchResponse(
       SearchResponse searchResponse,
       OpenSearchExprValueFactory exprValueFactory,
-      List<String> includes) {
+      List<String> includes,
+      boolean fieldTypeTolerance) {
```

Comment:
fieldTypeTolerance is not releated to OpenSearchResponse, Could we move this logic to OpenSearchExprValueFactory?

Reply:
I can look, but making this change could take hours (with updating the affected tests).

Reply:
Now it is passed through OpenSearchExprValueFactory